src/assets/markdown.scss

```diff
@@ -25,7 +25,14 @@
 
 	input:disabled + .checkbox-content {
 		opacity: 1 !important;
-		pointer-events: none !important;
+		cursor: default;
+
+		* {
+			cursor: default;
+		}
+		a {
+			cursor: pointer;
+		}
```

Comment:
```suggestion
```

?

Reply:
Then it would be a faulty pointer for plain text

Reply:
Shouldn't be handled upstream?

Reply:
true